### ISSUE TITLE
Virtual Camera does not work in Gotomeeting

### ISSUE THREAD

**Issue by CoreyBurger** [obsproject/obs-studio#4088]
## Platform
Operating system and version: Windows 10
OBS Studio version: 26.1.1

## Expected Behavior
Gotomeeting should be able to see and use the Virtual Camera

## Current Behavior
Gotomeeting says "Your webcam is already in use. Please close all other applications using your webcam and try again"

## Steps to Reproduce

1. Start OBS
2. Start gotomeeting
3. Try and select OBS Virtual Camera, get error message
4.

## Additional information
This may be the same bug #3545, but it may not be

**Comment by abrothman** [comment]
I can reproduce this error in Windows 10 with 26.1.1

**Comment by DoD-Buko** [comment]
I have same problem. OBS is new intall.

**Comment by abrothman** [comment]
A workaround until this is fixed is to use the old solution, the [OBS Virtualcam](https://obsproject.com/forum/threads/obs-virtualcam.121448/) plugin.

Note that the Windows installer tries to install to `\Program Files\obs-studio`, which doesn't work. I had to manually change it to `\Program Files\obs-studio\obs-plugins`.

**Comment by Fenrirthviti** [comment]
I have just tested the installer and looked at the script (as I wrote it) and this is not accurate. The correct location to select is the base obs-studio installation directory. I am honestly confused how it could be working at all selecting obs-plugins as you would get a duplicate but nested file structure from the root folder.

**Comment by abrothman** [comment]
You're correct - my apologies, and thanks for your work!

**Comment by DoD-Buko** [comment]
I used the zip file and copied into the directory.
but still does not work
:(

**Comment by abrothman** [comment]
@DoD-Buko - Note that you have to start the (plugin) virtual camera from the Tools menu.

**Comment by DoD-Buko** [comment]
I started the plugin via tools and then gotomeeting

**Comment by dcowan56** [comment]
As of 19 Oct 2021, this is still an issue in version 27.1.3

**Comment by MBBAD** [comment]
There is no problem with OBS and GoToMeeting. It is just a question of sequence:
1) keep OBS closed before starting the meeting!!
2) start the meeting and choose OBS camera -> you will see the symbol of OBS 
3) now you can start OBS and start the virtual camera
that works

**Comment by RytoEX** [comment]
Please re-test this in OBS Studio 28.1.1 (or newer) and confirm if this is still an issue. We've deployed many changes and fixes for the Virtual Camera on Windows, so we think this is fixed, but we would like to confirm.

**Comment by MBBAD** [comment]
yes, it´s fixed

Herzliche Grüße

* Martin Braun*

Am Mi., 2. Nov. 2022 um 21:18 Uhr schrieb Ryan Foster <
***@***.***>:

**Comment by gxalpha** [comment]
Closing per above comment

**Comment by axunonb** [comment]
@MBBAD, @gxalpha  Sorry, cannot confirm. It still does not work with v28.1.2. 
Switching to OBS Virtual Camera in GotoMeeting just shows the hourglass, but no video. Looking at GotoMeeting settings, I see "Initializing the web cam failed".

**Comment by alreadyhaveanaccount** [comment]
It's still a problem (v28.1.2; Windows 11) GoTo Meeting doesn't show OBS symbol on start and later no video. Zoom, Teams, Skype ... no problem.

**Comment by RytoEX** [comment]
@axunonb 
We've reverted a commit that was reportedly breaking Webex, which may have also affected GoToMeeting. Please retest in a future release of OBS. If it is not fixed in the next release, please feel free to open a new Issue with an OBS log and Steps To Reproduce.
